### ISSUE TITLE
Can fluentd-gcp-scaler/Dockerfile run as a non root user?

### ISSUE THREAD

**Issue by xynova** [GoogleCloudPlatform/k8s-stackdriver#354]
Our compliance system is alerting about the fluentd-gcp-scaler running as root in our GKE clusters. This is automatically deployed by GKE so we cant really modify its deployment. Is it worth adding a default non root user in the Dockerfile?

https://github.com/GoogleCloudPlatform/k8s-stackdriver/blob/master/fluentd-gcp-scaler/Dockerfile

**Comment by x13n** [comment]
Yes, it shouldn't need to run as root. Feel free to send me a PR.

**Comment by mmosierteamvelocity** [comment]
Was this ever implemented? We need to run this container as non root in our GKE environments. @x13n

**Comment by mmosierteamvelocity** [comment]
@serathius

**Comment by mmosierteamvelocity** [comment]
Just to clarify, this pertains to this image: k8s.gcr.io/fluentd-gcp:1.30, correct?
